js/gemini.js

```diff
@@ -1414,11 +1413,20 @@ module.exports = class gemini extends Exchange {
         };
     }
 
-    async fetchDepositAddressesByNetwork (code, params = {}) {
+    async fetchDepositAddress (code, params = {}) {
```

Comment:
I don't think we should rename this method, since Gemini's endpoint returns multiple addresses in their response – we can index them by network: https://docs.gemini.com/rest-api/#get-deposit-addresses , or am I missing something?

Reply:
In it's current form I get this response `gemini fetchDepositAddressesByNetwork() requires a network parameter` so it does only return one network's address

Reply:
@carlosmiei Any opinion?